src/services/redisSessionStorage.ts

```diff
@@ -30,6 +38,7 @@ const sessionKeyPrefix = "SESSION-";
 const walletKeyPrefix = "WALLET-";
 const userSessionsSetKeyPrefix = "USERSESSIONS-";
 const sessionInfoKeyPrefix = "SESSIONINFO-";
+const blockedUserSetKey = "BLOCKEDUSERS";
```

Comment:
Could be compatibility issues with `BLOCKED USERS` key? Previously was `BLOCKED-USERS`

Reply:
Well, actually this has been used by [this operation](https://github.com/pagopa/io-functions-admin/pull/55/files#diff-42c51bd2e19fc8609eb1dce93761ee97R28) mostly.

Here we just checked for a user to be blocked, but I guess it never worked as it looked for `BLOCKED-USERS` not `BLOCKEDUSERS`.

Long story short, I think both are good but `BLOCKEDUSERS` has even fewer chances to incompability.